src/pid/PIDControllerRunner.java

```diff
@@ -0,0 +1,32 @@
+package pid;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.IMotorControllerEnhanced;
+
+public class PIDControllerRunner implements PIDRunner {
+  private IMotorControllerEnhanced motorControllerEnhancedPIDRunner;
+  private MotorControllerEnhancedPIDController motorControllerEnhancedPIDController;
```

Comment:
Actually really like you put the implementation here, as it's not sensible to use any other with this runner, so this is better in terms of type safety. Good job!